<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="2" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="CompteurVitesse">
        <property name="text">
         <string>Vitesse : 0 px/s</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="StartpushButton">
        <property name="text">
         <string>Commencer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PausepushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RedemarrerpushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Redémarrer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="layout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAjouter_Graphique">
    <property name="title">
     <string>Option Graphique</string>
    </property>
    <addaction name="action_ajouter"/>
   </widget>
   <widget class="QMenu" name="menuPersonalisation_voiture">
    <property name="title">
     <string>Personalisation voiture</string>
    </property>
    <addaction name="actionCouleur"/>
    <addaction name="actionCarac"/>
   </widget>
   <addaction name="menuAjouter_Graphique"/>
   <addaction name="menuPersonalisation_voiture"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_ajouter">
   <property name="text">
    <string>ajouter</string>
   </property>
  </action>
  <action name="actionCouleur">
   <property name="text">
    <string>Couleur Voiture</string>
   </property>
  </action>
  <action name="actionCarac">
   <property name="text">
    <string>Caractéristiques Voiture</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
